src/MICore/MICoreResources.resx

```diff
@@ -288,4 +288,10 @@ Error: {1}</value>
     <value>Unknown launchCompleteCommand value '{0}'. Expected values are 'exec-run', 'exec-continue' and 'None'.</value>
     <comment>{0} value</comment>
   </data>
+  <data name="Error_UnsupportedGdb" xml:space="preserve">
+    <value>GDB version {0} is not supported. Consider upgrading.
```

Comment:
AFAIK we have only seen this issue on Cygwin. Have you verified this bug occurs on other platforms (Ubuntu)? I don't think we want to put out this message for other platforms unless we know its a problem.

Reply:
I think you probably want to grab the 'Cygwin' part of the text also, and then change the unsupported version info to include that.

Reply:
I just verified that MI mode works on GDB 7.12 Ubuntu, so as far as I know it's only a MinGW/Cygwin issue. Added a check for MinGW/Cygwin.

Reply:
I agree with Gregg. either add the type of gdb or add something about not being supported on Windows ?